src/components/notifications/PushPlus.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+    <div class="mb-3">
+        <label for="pushPlus-sendkey" class="form-label">{{ $t("SendKey") }}</label>
+        <HiddenInput id="pushPlus-sendkey" v-model="$parent.notification.pushPlusSendKey" :required="true" autocomplete="new-password"></HiddenInput>
+    </div>
+    <a href="https://www.pushplus.plus/" target="_blank">LearnMore"</a>
```

Comment:
Here is how I think this should be localised instead.

```suggestion
    <i18n-t tag="div" keypath="More info on:" class="mb-3 form-text">
      <a href="https://www.pushplus.plus/" target="_blank">https://www.pushplus.plus/</a>
    </i18n-t>
```